website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+
+
+### DIA Documentation
+
+The [DIA documentation](https://jargon.sh/user/unece/DigitalIdentityAnchor/v/0.1.0/artefacts/readme/render) provides a technology-neutral definition of classes, properties and code lists in the DIA model.
+
+## Implementation Guidance
+
+This section provides sample JSON-LD snippets for each DIA component.
+
+### Digital Identity Anchor
+
+The Digital Identity Anchors is a Verifiable credential. Please refer to [DPP VC Guidance](DigitalProductPassport.md#verifiable-credential) for information about the use of the verifiable credentials data model for UNTP.  THe issuing party for the VC MUST be the authority that owns or operates the identity register. 
+
+### Registered Identity
+
+The registered identity class represents the registry member. For example, in a national business register, the registered identity would be one of the registered businesses.
+
+* `type` MUST contain constant value array `["RegisteredIdentity", "Identifier"]` which declares to linked data processors that this is a registered identity which is a sub-type of identity.
+* the `id` MUST contain the globally unique member identifier as maintained in the register.  Often this will be a web link to the member entry.
+* `name` is the human readable registered entity name and SHOULD match the registered name as it appears in the authoritative register.
+* `registeredId` contains the simple identifier value that is unique within the register (but may not be globally unique) for example the VAT registration number.
+* `idScheme` identifies the authoritative register.  If the identity scheme is registered with UN/CEFACT then the `idScheme.id` MUST match the `identityRegister.id` in the UN/CEFACT scheme register.
+* `registerType` is a coded value that allows verifiers to distinguish between different DIA [use cases](#use-cases)
+* `verifiedDIDList` contains an array of DIDs that the member has requested (and the registrar has verified) to be linked to the registeredId of the member.
+* `registrationScopeList` contains a list of URIs that define the scope of the member registration. The values are very specific to the register. For example a national business register would typically have a controlled vocabulary of entity types (eg [Australian Entity Types](https://abr.business.gov.au/Help/EntityTypeList))
+
+
+```json
+  "credentialSubject": {
+    "type": [
+      "RegisteredIdentity",
+      "Identifier"
+    ],
+    "id": "https://sample-register.gov/90664869327",
+    "name": "Sample business Ltd",
+    "registeredId": "90664869327",
+    "idScheme": {
+      "type": [
+        "IdentifierScheme"
+      ],
+      "id": "https://sample-register.gov",
+      "name": "Sample National Business Register"
+    },
+    "registerType": "Business",
+    "verifiedDIDList": [
```

Comment:
Inelegant. This should be `credentialSubject.id`

Reply:
also +1, I would like to see some real world exploration about the use case for a `verifiedDIDList`, there's alot to consider by going down this path

Reply:
There are real world scenarios (see https://github.com/uncefact/spec-untp/issues/37#issuecomment-2458213130) but I think we all agree for now that they are better handled with multiple DIAs rather than one DIA with a DID list.  Will update the model accordingly